docs/content/preview/explore/query-1-performance/query-diagnostics.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Query diagnostics
+linkTitle: Query diagnostics
+headerTitle: Query diagnostics
+headcontent: Export diagnostic information for analysis
+tags:
+  feature: tech-preview
+menu:
+  preview:
+    identifier: query-diagnostics
+    parent: query-tuning
+    weight: 500
+type: docs
+---
+
+Use query diagnostics to troubleshoot and analyze database query performance issues. You can capture and export detailed diagnostic information across multiple dimensions, and use this information to identify and resolve database query problems.
+
```

Comment:
The meaning of the first and last sentences is almost the same, making the first statement redundant. Rephrase?